### DISCUSSION TITLE
warning when a function is overwritten by an other one

### DISCUSSION THREAD

**Discussion by phsdv** [custom-components/pyscript#539] [Ideas]
It happened to me already several times that I have very similar functions, with only a trigger and or state that is different. In that case I copied the few lines of code, adjusted the trigger and state_active decorators and wonder why everything did not work as planned. As I did not change the name of the function the second one overwrote the first and the first would never work :-(

It would be nice to have some sort of warning that the function has been overwritten

**Comment by j-steve** [comment]
Filed a [feature request](https://github.com/custom-components/pyscript/issues/557) for this issue
